packages/react-components/react-tooltip/src/components/Tooltip/Tooltip.types.ts

```diff
@@ -117,13 +54,73 @@ export type TooltipProps = ComponentProps<TooltipSlots> &
       | (React.ReactElement & { ref?: React.Ref<unknown> })
       | ((props: TooltipTriggerProps) => React.ReactElement | null)
       | null;
+
+    /**
+     * Delay before the tooltip is hidden, in milliseconds.
+     *
+     * @default 250
+     */
+    hideDelay?: number;
+
+    /**
+     * Notification when the visibility of the tooltip is changing
+     */
+    onVisibleChange?: (
+      event: React.PointerEvent<HTMLElement> | React.FocusEvent<HTMLElement> | undefined,
+      data: OnVisibleChangeData,
+    ) => void;
+
+    /**
+     * Configure the positioning of the tooltip
+     *
+     * @default above
+     */
+    positioning?: PositioningShorthand;
+
+    /**
+     * (Required) Specifies whether this tooltip is acting as the description or label of its trigger element.
+     *
+     * * `label` - The tooltip sets the trigger's aria-label or aria-labelledby attribute. This is useful for buttons
+     *    displaying only an icon, for example.
+     * * `description` - The tooltip sets the trigger's aria-description or aria-describedby attribute.
+     * * `inaccessible` - No aria attributes are set on the trigger. This makes the tooltip's content inaccessible to
+     *   screen readers, and should only be used if the tooltip's text is available by some other means.
+     */
+    relationship: 'label' | 'description' | 'inaccessible';
+
+    /**
+     * Delay before the tooltip is shown, in milliseconds.
+     *
+     * @default 250
+     */
+    showDelay?: number;
+
+    /**
+     * Control the tooltip's visibility programatically.
+     *
+     * This can be used in conjunction with onVisibleChange to modify the tooltip's show and hide behavior.
+     *
+     * If not provided, the visibility will be controlled by the tooltip itself, based on hover and focus events on the
+     * trigger (child) element.
+     *
+     * @default visible
```

Comment:
I think this should be `@default false` rather than `@default visible` :)

Reply:
Thanks!